### ISSUE TITLE
Unable to download 3.3.1-arsenic

### ISSUE THREAD

**Issue by testphreak** [SeleniumHQ/docker-selenium#430]
## Meta -
Image(s):  
node-chrome, hub
Docker Version:  
3.3.1-arsenic
OS: 
Mac, Linux

## Expected Behavior -
I should be able to download the images as mentioned in the README.md
## Actual Behavior -
```
$ docker run -d -p 4444:4444 --name selenium-hub selenium/hub:3.3.1-arsenic
Unable to find image 'selenium/hub:3.3.1-arsenic' locally
Pulling repository docker.io/selenium/hub
docker: Tag 3.3.1-arsenic not found in repository docker.io/selenium/hub.
See 'docker run --help'.
```

**Comment by testphreak** [comment]
@ddavison are the images for 3.3.1-arsenic not on Docker Hub yet?

**Comment by ddavison** [comment]
they aren't because the build is botched.  gotta figure out what's going on.  while at the conference - i'll be figure out what the heck happened.

**Comment by enroxorz** [comment]
+1 Thank you

**Comment by testphreak** [comment]
Sounds good @ddavison

**Comment by DerFlatulator** [comment]
Is there a version to roll back to that didn't have the 3.3.0 connectivity issues?

**Comment by tparikka** [comment]
@DerFlatulator If you're referring to #425 I have had great success with selenium/hub:3.2.0 and selenium/node-chrome:3.2.0 (as well as node-chrome-debug). However, depending on how heavy you are hitting your Grid you will want to be aware of docker/for-win#426, which the Docker for Windows team is working on - I've had to stop using containers for the moment because under heavy load the VPNKit component of Docker for Windows may fail and require a restart of the entire Docker service to resolve. The workaround for the moment seems to be to use docker-machine to create full machines until the issue can be fully resolved.

**Comment by DerFlatulator** [comment]
@tparikka Thanks for that. I'll give 3.2.0 a shot while the 3.3.1 image is being sorted out.

**Comment by beniamin-kis** [comment]
@ddavison any news yet?

**Comment by setiseta** [comment]
There is also a new version of chromedriver: 2.29

**Comment by beniamin-kis** [comment]
In the meantime, I cloned the repo and build the images myself (with VERSION=local)

**Comment by tparikka** [comment]
@beniamin-kis What environment did you use to build? I gave it a shot with Cygwin on Windows 10 but I ran into issues first with EOL encoding issues in generate.sh and then got this error:

/bin/sh: 1: /opt/selenium/generate_config: not found

If it's possible to build on Windows with Cygwin I'd prefer that, but if it's not possible or it's just better to build on Linux I can spin up a VM and do it that way.

**Comment by beniamin-kis** [comment]
uhm, I used a mac. I'm not sure how could you do it on windows. maybe use the new bash for Windows?

**Comment by setiseta** [comment]
@beniamin-kis what source you have used? in this repo or changed something? maybe you can provide this image's temporary on dockerhub until the official gets updated?

**Comment by beniamin-kis** [comment]
I've just cloned the master branch of this repo, changed line 27 @ /NodeChrome/Dockerfile.txt:

 #==================
 # Chrome webdriver
 #==================
-ARG CHROME_DRIVER_VERSION=2.28
+ARG CHROME_DRIVER_VERSION=2.29

and then I've ran:

$ VERSION=local make build

**Comment by ddavison** [comment]
fixed in 3.3.1-cesium 👍
